Dnn.AdminExperience/ClientSide/Servers.Web/src/services/smtpServerService.js

```diff
@@ -17,7 +17,7 @@ const sendTestEmail = function (parameters) {
     return serviceFramework.post(getControllerName(), "SendTestEmail", parameters);
 };
 
-const getOAuthProviders = function(){
+const getOAuthProviders = function () {
```

Comment:
This appears to be unrelated

```suggestion
const getOAuthProviders = function(){
```

Reply:
Yes, unnecessary, but this eliminates a warning and makes the code consistent with the other 3 functions above.